addons/mail/static/src/core/web/activity_menu.js

```diff
@@ -55,6 +60,14 @@ export class ActivityMenu extends Component {
             // So, duplicates are faking the count and "Load more" doesn't show up
             force_search_count: 1,
         };
+        if (group.model === "mail.activity") {
+            this.action.doAction("mail.mail_activity_without_access_action", {
+                additionalContext: {
+                    active_ids: group.activity_ids,
+                },
+            });
```

Comment:
Could you add a protection if "mail.mail_activity_without_access_action" does not exist ? If we add this code, people might encounter a crash when testing the feature during OXP. Just fallback on any standard activity action (even if not perfect, better than a crash)